doc_src/faq.hdr

```diff
@@ -220,6 +221,11 @@ rm -f fish fish_indent
 
 Fish reserves the <a href="http://www.unicode.org/faq/private_use.html">Unicode private-use character range</a> from U+F600 thru U+F73F for internal use. Any attempt to feed characters in that range to fish will result in them being replaced by the Unicode "replacement character" U+FFFD. This includes both interactive input as well as any file read by fish (but not programs run by fish).
 
+<hr>
+\section faq-third-party Where can I find 3rd-party extensions?
+
+The fish user community extends fish in unique and useful ways that aren't always appropriate for inclusion in the core fish distribution. You can find those extensions in various third-party repositories and fish package managers. These include <a href="https://github.com/fisherman/fisherman">Fisherman</a>, <a href="https://github.com/tuvistavie/fundle">Fundle</a>, <a href="https://github.com/oh-my-fish/oh-my-fish">Oh My Fish</a>, <a href="https://github.com/justinmayer/tacklebox">Tacklebox</a>. This is not an exhaustive list and the fish project has no opinion regarding the merits of the various options.
```

Comment:
how about "...find those extensions, including prompts, themes and useful functions, in various third-party repositories"?